java/src/jmri/jmrix/loconet/LnStringIOManager.java

```diff
@@ -49,6 +52,25 @@ public void dispose() {
         super.dispose();
     }
 
+    @Override
+    @Nonnull
+    public StringIO provideStringIO(@Nonnull String sName) throws IllegalArgumentException {
+        return new DefaultStringIO(sName);
```

Comment:
I strongly suggest throwing an `IllegalArgumentException` here for now.

`throw new IllegalArgumentException("Generic LocoNet StringIOs are not yet supported");`

Reply:
> I strongly suggest throwing an IllegalArgumentException here for now.

I changed it three places where the LnStringIoManager was created a StringIO